packages/sdk/src/evm/common/feature-detection.ts

```diff
@@ -196,7 +195,9 @@ export function extractFunctionsFromAbi(
     const promise = out ? `: Promise<${out}>` : `: Promise<TransactionResult>`;
     const signature = `contract.call("${f.name}"${fargs})${promise}`;
     parsed.push({
+      // @ts-ignore we know AbiTypeBaseSchema.name is not going to be undefined since we're doing `.default("")`
```

Comment:
i think you could avoid this if you wrap `f.inputs` into a `AbiInput[]` type, and put a default value for name in each of them.

nbd though i think this should be mostly fine